redact/main.py

```diff
@@ -206,6 +237,15 @@ def redact_folder(
         auto_delete_input_file=auto_delete_input_file,
     )
 
+    for file_path, exception in exceptions.items():
+        log.info(
```

Comment:
Shouldn't we do at least warn level here?

Reply:
done